pkg/cmd/container/inspect.go

```diff
@@ -47,12 +50,45 @@ func Inspect(ctx context.Context, client *containerd.Client, containers []string
 			log.L.Error(formatErr)
 		}
 	}
+
 	return err
 }
 
 type containerInspector struct {
-	mode    string
-	entries []interface{}
+	mode        string
+	size        bool
+	snapshotter snapshots.Snapshotter
+	entries     []interface{}
+}
+
+func resourceTotal(ctx context.Context, snapshotter snapshots.Snapshotter, resourceID string) (snapshots.Usage, snapshots.Usage, error) {
+	var first snapshots.Usage
+	var total snapshots.Usage
+	next := resourceID
+
+	for next != "" {
```

Comment:
This could be written as `for next := resourceID; next != ""; next = info.Parent` ?

Reply:
That seems fair to me. Will update.

Reply:
Fair.
Will update.